frontend/src/services/ga.service.ts

```diff
@@ -19,9 +19,10 @@ export const GA_USER_EVENTS = {
   COMPLETE_DUPLICATE: 'Complete campaign duplication',
 }
 
-export function initializeGA() {
+export function initializeGA(options?: ReactGA.InitializeOptions) {
   ReactGA.initialize(GA_TRACKING_ID, {
```

Comment:
This currently throws a warning as GA_TRACKING_ID is not provided. Might be good to mock the entire ga service during tests. This way we also don't need to pass `testMode: true` to `AuthContextProvider`.

Reply:
I've decided to inject the options for ReactGA using an exported variable from `config.ts`. So now `initializeGA` looks like this: https://github.com/opengovsg/postmangovsg/pull/1086/commits/01718ba811a86a35309cb370d4076ffb63de37b9#diff-1791205ed2d87458629a79c784c32fb6f18ab0a797f42c8635b11b589ec72371R23